src/renderer/store/modules/history.js

```diff
@@ -36,6 +42,15 @@ const actions = {
     }
   },
 
+  async selectSort({ commit }, sort) {
+    try {
+      const order = sort
+      commit('setSortOrder', order)
+    } catch (errMessage) {
+      console.error(errMessage)
+    }
+  },
+
```

Comment:
As this action is just a plain wrapper, there isn't any benefit to using it compared to just calling the mutation directly, so it's better to just call the mutation directly in this case.
```suggestion
```